### PR TITLE
Autosuggest - categories

### PR DESCRIPTION
**Pull request by Adamik10**
### Ticket
https://reload.atlassian.net/browse/DDFSOEG-101

### Description
This PR introduces:
- boxed text
- autosuggest-categories

### Screenshots
image
image

image
image

### CODE REVIEW

File: src/stories/Library/spacing/spacing.scss [hunk 1]
```diff
@@ -12,6 +13,7 @@ $s-215: 215;
 
 $s-xs: 4px;
 $s-sm: 8px;
+$s-smd: 10px;
```
Comment (kasperg) [inline src/stories/Library/spacing/spacing.scss:16]: Consider if we should ask Dept whether this 10px size is intentional. We use a 4/8-based size system where 10 does not really fit in.

Comment (Adamik10) [reply src/stories/Library/spacing/spacing.scss:16]: I think we could even go with a 12px option (8+4) - but there really is nothing between 8 and 16 pixels right now.
But I will ask in Figma :)

File: src/stories/Library/autosuggest-text/AutosuggestText.stories.tsx [hunk 2]
```diff
@@ -13,6 +13,16 @@ export default {
       defaultValue: ["Item one", "Item two"],
       control: { type: "array" },
     },
+    withCategory: {
+      name: "Show category?",
+      defaultValue: false,
+      control: { type: "boolean" },
+    },
```
[the remark this answers is not in the document]
Comment (Adamik10) [reply src/stories/Library/autosuggest-text/AutosuggestText.stories.tsx]: That's perfect! Let's get rid of unnecessary bloating. 👍 💯

**Review by yepzdk — APPROVED** [review]
Looks good

**Pull request merged**